### PR TITLE
Update template with necessary files

### PR DESCRIPTION
**Pull request by TarasSpashchenko**
folio-spring-base dependency version bumped up to 4.1.0

update template with necessary files:
* main spring boot class
* basic test class
* Dockerfile
* replace application .properties-files with .yaml-files

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -24,7 +24,7 @@
 
   <properties>
     <java.version>11</java.version>
-    <folio-spring-base.version>4.0.0</folio-spring-base.version>
+    <folio-spring-base.version>4.1.0</folio-spring-base.version>
```
Comment (julianladisch) [inline pom.xml:37]: folio-spring-base 4.1.0 is based on spring-boot-starter-parent 2.7, therefore spring-boot-starter-parent should be bumped from 2.6.4 to 2.7.1

Comment (TarasSpashchenko) [reply pom.xml:37]: Thank you. Fixed.

File: pom.xml [hunk 2]
```diff
@@ -35,6 +46,33 @@
       <artifactId>folio-spring-base</artifactId>
       <version>${folio-spring-base.version}</version>
     </dependency>
+	<dependency>
```
Comment (julianladisch) [inline pom.xml]: indentation

**Review by dmtkachenko — APPROVED** [review]

File: pom.xml (hunk 2 above)
Comment (TarasSpashchenko) [reply pom.xml]: Done

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,11 @@
+FROM folioci/alpine-jre-openjdk17:latest
+
+# Copy your fat jar to the container provide the actual name for your fat jar file for example mod-notes-fat.jar
+ENV APP_FILE mod-spring-tempale.jar
```
Comment (dcrossleyau) [inline Dockerfile]: More typos

Comment (dmtkachenko) [reply Dockerfile]: updated

**Pull request merged**